src/components/standup/standupContainer.module.scss

```diff
@@ -0,0 +1,157 @@
+$theme-CornFlowerBlue: #1c64f2;
+$theme-Biscay: #1c325e;
+$theme-white: #ffffff;
+$theme-FrenchRose: #f1416c;
+$theme-FunBlue: #1e429f;
+$theme-CeriseRed: #d61f69;
+$theme-PaleSky: #6b7280;
+$theme-Mischka: #d1d5db;
+$theme-AthensGray: #e5e7eb;
+$radius: 0.5rem;
+
+.standupContainer {
+    display: flex;
+    flex-direction: column;
+    margin-top: 1.875rem;
+    width: 100%;
+    align-items: center;
+    justify-content: center;
+}
+
+.standupBanner {
+    background-color: $theme-Biscay;
+    width: 83.6875%;
+    height: 12%;
+    color: $theme-white;
+    padding: 1.5rem 0 1.5rem 3.0625rem;
+    border-radius: $radius;
+}
+
+.standupBanner p {
+    font-family: 'Poppins', sans-serif;
+    font-size: 1.25rem;
+    font-weight: 600;
+}
+
+.standupBanner p span {
+    color: $theme-FrenchRose;
+    text-decoration: underline;
+    margin: 0 0.25rem;
+}
+
+.standupUpdateContainer {
+    width: 50.63625%;
+    margin-top: 3.125rem;
+}
+
+.standupForm {
+    margin-top: 2.1875rem;
+}
+
+.standupUpdateContainer h1 {
+    color: $theme-FunBlue;
+    font-family: 'Inter', sans-serif;
+    font-size: 2.2rem;
+}
+
+.yesterdayUpdate,
+.todayUpdate,
+.blockerUpdate {
+    margin-bottom: 1.25rem;
+}
+
+.yesterdayUpdate label,
+.todayUpdate label,
+.blockerUpdate label {
+    color: $theme-PaleSky;
+    font-size: 1.4rem;
+    margin-top: 2.5rem;
+    margin-bottom: 0.625rem;
+    font-family: 'Inter', sans-serif;
+}
+
+.inputFlield {
+    height: 58px;
+    width: 100%;
+    border-radius: $radius;
+    border: 1px solid $theme-Mischka;
+    background: $theme-white;
+    padding: 1rem;
+    font-weight: 400;
+    font-size: 0.875rem;
+    line-height: 150%;
+    font-family: 'Inter', sans-serif;
+    margin-top: 0.8125rem;
+}
+
+.inputFlield:focus {
+    border: 1px solid $theme-CornFlowerBlue;
+    outline: none;
+}
+
+.submitButton {
```

Comment:
Can you add the hover effect when user filled the form and also highlight the button

Reply:
you can ask to others too in standup this